canonical-web-and-design/ubuntu.com
# Course mappings and URLs

## Done

- Update mappings file to contain the newly created course IDs
- Update take URLs
- Add test coverage for cube endpoints

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube/microcerts
-  Make sure you see the 15 certifications in the state they should be for your account
